images/kubernetesonarm/elasticsearch/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM larmog/armhf-alpine-java:jdk-8u73
+
+ENV VERSION 2.2.1
```

Comment:
Please add `ELASTICSEARCH_VERSION` to `images/version.sh`, `COPY version.sh /`
Copy it to current dir via a `build.sh` file first.
Then `RUN . /version.sh && ... `
The `go` image is set up this way, you may find inspiration there (or I may do it as a followup)